app/assets/stylesheets/application.css

```diff
@@ -20,4 +20,6 @@
  *= require product_slickgrid
  *= require kubernetes-topology-graph/topology-graph
  *= require jquery-ui-miq-theme/jquery-ui-1.9.2.custom
+ *= require angular-patternfly
+ *= require dashboard
```

Comment:
Agreed with abonas here, dashboard might be too generic, especially given the first page you load in ManageIQ is also called dashboard